packages/babel-helpers/src/helpers/objectSpread.ts

```diff
@@ -26,7 +26,11 @@ export default function _objectSpread(target: object) {
       );
     }
     ownKeys.forEach(function (key) {
-      defineProperty(target, key, source[key as keyof typeof source]);
+      defineProperty(
+        target,
+        toPropertyKey(key),
```

Comment:
Here `key` is already the right type -- could you change line 18 to use `(string | symbol)[]` instead of adding a call to `toPropertyKey`?